src/components/Footer/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { FaHeart, FaReact } from "react-icons/fa"
+import s from "./footer.module.scss"
+
+export default function Footer() {
+  const year = new Date().getFullYear()
+
+  return (
+    <footer>
+      <span className={s.copyright}>
+        Made with{" "}
+        <span className={s.icon}>
+          <FaHeart className="pulse" />{" "}
+        </span>{" "}
+        and{" "}
+        <span className={s.icon}>
+          <FaReact className="pulse-delay" />
+        </span>{" "}
+      </span>
```

Comment:
When using icons as text, the text does not make when read by screen readers (used mostly by blind people).

"Made with pulse and pulse delay"

A possible fix is to add `aria-label` for each icon.

<FaHeart className="pulse" aria-label="love" />